buffer-crc32 required as a dependency for Jeff

I had a problem using jeff initially and it reported an error about "Error: Cannot find module 'buffer-crc32'".
Looking through the code I found a require('buffer-crc32'); call in the lib/CanvasRenderer/SwfImgRenderer/PngWriter.js file. To fix this I had to go either a "npm install buffer-crc32" or "npm install -g buffer-crc32". To fix it permanently I think it would be better to add buffer-crc32 as a dependency in package.json. I have created a PR for this change

--- a/.gitignore
+++ b/.gitignore
@@ -1,2 +1,3 @@
 test
 npm-debug.log
+node_modules

--- a/package.json
+++ b/package.json
@@ -1,34 +1,33 @@
 {
-	"name": "jeff",
-	"description": "Extracts Json meta-data and images from a SWF Flash file.",
-	"version": "0.1.7",
-	"private": false,
-	"author": {
-		"name": "Brice Chevalier",
-		"email": "bchevalier@wizcorp.jp"
-	},
-	"bin": {
-		"jeff": "./bin/jeff"
-	},
-	"main": "./lib/index.js",
-	"dependencies": {
-		"canvas": "1.1.6",
-		"glob": "4.3.1",
-		"image-optim": "0.3.0",
-		"commander": "2.2.0",
-		"fs-extra": "0.8.1",
-		"async": "0.2.10",
-		"js-beautify": "1.4.2",
-		"node-pngquant-native": "0.0.12"
-	},
-	"repository": {
-		"type": "git",
-		"url": "https://github.com/Wizcorp/jeff.git"
-	},
-	"licenses": [
-		{
-			"type": "MIT",
-			"url": "https://github.com/Wizcorp/jeff/blob/master/LICENSE"
-		}
-	]
+    "name": "jeff",
+    "description": "Extracts Json meta-data and images from a SWF Flash file.",
+    "version": "0.1.7",
+    "private": false,
+    "author": {
+        "name": "Brice Chevalier",
+        "email": "bchevalier@wizcorp.jp"
+    },
+    "bin": {
+        "jeff": "./bin/jeff"
+    },
+    "main": "./lib/index.js",
+    "dependencies": {
+        "async": "0.2.10",
+        "buffer-crc32": "^0.2.5",
+        "canvas": "1.1.6",
+        "commander": "2.2.0",
+        "fs-extra": "0.8.1",
+        "glob": "4.3.1",
+        "image-optim": "0.3.0",
+        "js-beautify": "1.4.2",
+        "node-pngquant-native": "0.0.12"
+    },
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/Wizcorp/jeff.git"
+    },
+    "licenses": [{
+        "type": "MIT",
+        "url": "https://github.com/Wizcorp/jeff/blob/master/LICENSE"
+    }]
 }

